test/helpers/position_assertions.cc

```diff
@@ -58,6 +58,11 @@ template <typename T> bool isDuplicateDiagnostic(string_view filename, T *assert
 template <typename T>
 void reportMissingError(const string &filename, const T &assertion, string_view sourceLine, string_view errorPrefix,
                         bool missingDuplicate = false) {
+    // Skip error message checking when running with Prism.
+    if (sorbet::test::parser == realmain::options::Parser::PRISM) {
+        return;
+    }
+
```

Comment:
We've been ignoring error messages from the parser in order to focus on tests that fail because of parse tree differences. We will come up with a system to address the differences in error messages before merging but we haven't gotten there yet.

Reply:
Reading through how this is used in `test/BUILD`, I'm a bit less concerned: the `--parser=prism` flag is only set for tests in the prism test corpus, and otherwise defaults to the `original` parser.